kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/KyuubiCli.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.ctl
+
+import java.util.{NoSuchElementException, StringJoiner}
+
+import scala.collection.JavaConverters._
+
+import com.beust.jcommander.{DefaultUsageFormatter, JCommander, MissingCommandException, ParameterException}
+
+import org.apache.kyuubi.Logging
+import org.apache.kyuubi.ctl.commands.common._
+import org.apache.kyuubi.ctl.commands.common.ServiceType.ServiceType
+import org.apache.kyuubi.ctl.commands.config.ConfigCommandGroup
+import org.apache.kyuubi.ctl.commands.engine._
+import org.apache.kyuubi.ctl.commands.server._
+
+class KyuubiCli extends Logging {
+
+  private lazy val commands = Map(
+    ServiceType.engine -> new EngineCommandGroup,
+    ServiceType.server -> new ServerCommandGroup,
+    ServiceType.config -> new ConfigCommandGroup
+  )
+
+  def printAllCommandUsage(): Unit = {
+    // scalastyle:off println
+    val sj = new StringJoiner("|", "<", ">")
+    ServiceType.values.foreach(v => sj.add(String.valueOf(v)))
+    println(s"Usage: kyuubi-ctl ${sj.toString} [actions] [options] \n")
```

Comment:
`${sj.toString}` => `$sj`

Reply:
`ServiceType.values.mkString("|", "<", ">")` result is `|server<engine<config>`